compiler/qsc/src/interpret/circuit_tests.rs

```diff
@@ -39,7 +39,7 @@ fn empty() {
     );
 
     let circ = interpreter
-        .circuit(CircuitEntryPoint::EntryPoint)
+        .circuit(CircuitEntryPoint::EntryPoint, false)
```

Comment:
I could not find any tests which pass `true` for simulation. Are there test cases we should add?

Reply:
Good point - will add.